src/components/grid-view/GridView.js

```diff
@@ -98,6 +113,7 @@ class GridView extends React.Component<Props> {
                 width={width}
                 gridClassName="bdl-GridView-body"
                 rowClassName="bdl-GridView-tableRow"
+                rowStyle={{ paddingRight: 0, width: '100%' }}
```

Comment:
Setting these styles seems to be necessary since we are altering `.bdl-GridView-body`.  (These style attributes can't be put on the class, because ReactVirtualized puts its own styles on the row elements, which would override what was set on `bdl-GridView-tableRow`.)

Reply:
This line is unnecessary if we don't alter `.bdl-GridView-body`, and just put all the margins/padding on the GridViewSlots themselves.

Reply:
Can we move it to CSS if we use higher specificity or `!important`? If not, this seems okay, but you could make it a constant if you really want.